code401challenges/src/main/java/linkedList/LinkedList.java

```diff
@@ -0,0 +1,64 @@
+// Author: Jack Kinne
+// challenge by: Codefellows
+package linkedList;
+
+public class LinkedList<T> {
+    private Node head;
+    private int nodeCount = 0;
+
+
```

Comment:
`Within your LinkedList class, include a head property. Upon instantiation, an empty Linked List should be created.` - the way you have this set up right now, never lets you have an empty LinkedList